.github/workflows/build-and-test.yaml

```diff
@@ -20,6 +20,13 @@ name: Check
 
 on:
   pull_request:
+    paths-ignore:
```

Comment:
Adding this will skip the checks in some cases and the `gateway` job below won’t succeed, in those cases pull requests are not able to be merged. Can you do this by leveraging the same way as you did in https://github.com/apache/skywalking-cli/blob/9ad35d097e8966ac9324f1d570b1a2d264b38ca1/.github/workflows/go.yml#L33 ?

Reply:
ok